tests/apis/platformvm/api.test.ts

```diff
@@ -1403,42 +1407,43 @@ describe("PlatformVMAPI", (): void => {
 
       serialzeit(tx1, "ExportTx")
     })
+
     /*
-        test('buildAddSubnetValidatorTx', async (): Promise<void> => {
-          platformvm.setFee(new BN(fee));
-          const addrbuff1 = addrs1.map((a) => platformvm.parseAddress(a));
-          const addrbuff2 = addrs2.map((a) => platformvm.parseAddress(a));
-          const addrbuff3 = addrs3.map((a) => platformvm.parseAddress(a));
-          const amount:BN = new BN(90);
-
-          const txu1:UnsignedTx = await platformvm.buildAddSubnetValidatorTx(
-            set,
-            addrs1,
-            addrs2,
-            nodeID,
-            startTime,
-            endTime,
-            PlatformVMConstants.MINSTAKE,
-            new UTF8Payload("hello world"), UnixNow()
-          );
-
-          const txu2:UnsignedTx = set.buildAddSubnetValidatorTx(
-            networkID, bintools.cb58Decode(blockchainID),
-            addrbuff1,
-            addrbuff2,
-            NodeIDStringToBuffer(nodeID),
-            startTime,
-            endTime,
-            PlatformVMConstants.MINSTAKE,
-            platformvm.getFee(),
-            assetID,
-            new UTF8Payload("hello world").getPayload(), UnixNow()
-          );
-          expect(txu2.toBuffer().toString('hex')).toBe(txu1.toBuffer().toString('hex'));
-          expect(txu2.toString()).toBe(txu1.toString());
-
-        });
+    test('buildAddSubnetValidatorTx', async (): Promise<void> => {
+      platformvm.setFee(new BN(fee));
+      const addrbuff1 = addrs1.map((a) => platformvm.parseAddress(a));
+      const addrbuff2 = addrs2.map((a) => platformvm.parseAddress(a));
+      const addrbuff3 = addrs3.map((a) => platformvm.parseAddress(a));
+      const amount:BN = new BN(90);
+
+      const txu1:UnsignedTx = await platformvm.buildAddSubnetValidatorTx(
+        set,
+        addrs1,
+        addrs2,
+        nodeID,
+        startTime,
+        endTime,
+        PlatformVMConstants.MINSTAKE,
+        new UTF8Payload("hello world"), UnixNow()
+      );
+
+      const txu2:UnsignedTx = set.buildAddSubnetValidatorTx(
+        networkID, bintools.cb58Decode(blockchainID),
+        addrbuff1,
+        addrbuff2,
+        NodeIDStringToBuffer(nodeID),
+        startTime,
+        endTime,
+        PlatformVMConstants.MINSTAKE,
+        platformvm.getFee(),
+        assetID,
+        new UTF8Payload("hello world").getPayload(), UnixNow()
+      );
+      expect(txu2.toBuffer().toString('hex')).toBe(txu1.toBuffer().toString('hex'));
+      expect(txu2.toString()).toBe(txu1.toString());
+    });
```

Comment:
Just confirming, do we want this test commented out?

Reply:
It was commented out before, just formatting